flowzone.yml

```diff
@@ -1469,7 +1469,12 @@ jobs:
   website_build:
     name: Build website using Docusaurus
     runs-on: ${{fromJSON(inputs.runs_on)}}
-    needs: [project_types, versioned_source]
+    env:
+     CF_BRANCH: github.event.pull_request.head.ref
```

Comment:
This job is not currently limited to pull requests, it will also run on push of tags (if enabled at the top level).

So we should either check that the event type is a pr, or decide what to do if someone has tagged a branch since `github.event.pull_request...` will not exist in that case.

Reply:
I could add a check on this line @klutchell 
https://github.com/product-os/flowzone/pull/342/files#diff-d899bb0fa13938dbb09e245516c63598abe5837797ab4bdf9860e7f1b70f3cadR1479

Don't know much about tags - but I guess I could check that type = PullRequestEvent?
https://docs.github.com/en/developers/webhooks-and-events/events/github-event-types